climada/hazard/tc_tracks_synth.py

```diff
@@ -37,43 +37,50 @@
+    temporal resolution, which can be achieved using equal_timestep().
+    max_dspeed_rel and autocorr_dspeed control the spread along the track ('what distance
+    does the track run for'), while max_ddir and autocorr_ddir control the spread
+    perpendicular to the track movement ('how does the track diverge in direction').
+    max_dspeed_rel and max_ddir control the amplitude of perturbations at each track
+    timestep but perturbations may tend to compensate each other over time, leading to
+    a similar location at the end of the track, while autocorr_dspeed and autocorr_ddir
+    control how these perturbations persist in time and hence the amplitude of the
+    perturbations towards the end of the track.
 
     The object is mutated in-place.
 
     Parameters:
         tracks (TCTracks): See `climada.hazard.tc_tracks`.
         ens_size (int, optional): number of ensemble members per track.
-            Default 9.
-        ens_amp0 (float, optional): amplitude of max random starting point
-            shift in decimal degree (longitude and latitude). Default: 1.5
-        ens_amp (float, optional): amplitude of random walk wiggles in
-            decimal degree (longitude and latitude). Default: 0.1
-        max_angle (float, optional): maximum angle of variation. Default: pi/10.
-            - max_angle=pi results in undirected random change with
-                no change in direction;
-            - max_angle=0 (or very close to 0) is not recommended. It results
-                in non-random synthetic tracks with constant shift to higher latitudes;
-            - for 0<max_angle<pi/2, the change in latitude is always toward
-                higher latitudes, i.e. poleward,
-            - max_angle=pi/4 results in random angles within one quadrant,
-                also with a poleward bias;
-            - decreasing max_angle starting from pi will gradually increase
-                the spread of the synthetic tracks until they start converging
-                towards the result of max_angle=0 at a certain value (depending
-                on length of timesteps and ens_amp).
+            Default: 9.
+        max_shift_ini (float, optional): amplitude of max random starting point
+            shift in decimal degree (up to +/-max_shift_ini for longitude and latitude).
+            Default: 0.75.
+        max_dspeed_rel (float, optional): amplitude of translation speed
+            perturbation in relative terms (e.g., 0.2 for +/-20%). Default: 0.3.
+        max_ddir (float, optional): amplitude of track direction (bearing angle) perturbation
+            per hour, in radians. Default: pi/180.
+        autocorr_dspeed (float, optional): autocorrelation of translation speed perturbation
+            at a lag of 1 hour. Default: 0.85.
+        autocorr_ddir (float, optional): autocorrelation of translational direction perturbation
+            at a lag of 1 hour. Default: 0.85.
```

Comment:
Since you are changing so much in this docstring anyways, maybe you can take the time to convert it to the NumPy style. According to CLIMADA's coding conventions, docstrings should follow the NumPy style. From one of the developer meetings, I remember that, while older code in CLIMADA is still allowed to contain google-style docstrings, new code is required to follow the new convention. Old docstrings should be converted to the new convention whenever they are touched.